File: C/maopao.c
#include <stdio.h> 

//冒泡算法
/*
** 比较相邻的两个数，大的往后放，小的放在前 
*/ 
int main(){
	int i,j;
	int arr[10];
	int temp;
	
	//接受用户的输入 
	for(i=0; i<10; i++){
		scanf("%d", &arr[i]);
	}
	
	//输出数组
	for(i=0; i<10; i++){
		printf("%d ", arr[i]);
		if(i==4){
			printf("\n");
		}
	} 
	
	//排序
	for(i=0; i<10; i++){
		for(j=0; j<9; j++){
			if(arr[j]>arr[j+1]){
				temp = arr[j];
				arr[j] = arr[j+1];
				arr[j+1] = temp;
			}
		}
	} 
	
	printf("\n");
	
	//输出数组
	for(i=0; i<10; i++){
		printf("%d ", arr[i]);
		if(i==4){
			printf("\n");
		}
	} 
	
	return 0;
	
}

File: C/maopao.php
<?php
/*
 * 比较相邻的两个数，小的往前交换，大的往后交换
 */
function maopao($arr){
	if(!is_array($arr)){
		return;
	}
	$length = count($arr);
	for($i=0; $i<$length; $i++){
		for($j=0; $j<9; $j++){
			if($arr[$j]>$arr[$j+1]){
				$temp = $arr[$j];
				$arr[$j] = $arr[$j+1];
				$arr[$j+1] = $temp;
			}
		}
	}
	
	return $arr;
}

$arr = array(
		3, 10, 2, 6, 9, 4, 1, 7, 8, 5
	);

print_r(maopao($arr));
?>